### PR TITLE
Features/create font style component

### PR DESCRIPTION
**Pull request by ridwanraji**
## Description

- Create font style component
- Mock layout in Designer.vue to view fontStyle

## Related Issue or Trello Card

[Trello card](https://trello.com/c/WiuqepF3/21-integrate-default-slide-template-with-authoring-view)

## How Has This Been Tested?

- [x] Unit test
- [ ] NightWatch test
- [ ] Coverage testing
- [x] Manual testing

## Screenshots (if appropriate):
screen shot 2018-03-02 at 12 05 55 am
screen shot 2018-03-02 at 12 07 43 am
screen shot 2018-03-02 at 12 07 08 am
screen shot 2018-03-02 at 12 06 59 am
screen shot 2018-03-02 at 12 06 39 am
screen shot 2018-03-02 at 12 06 28 am

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ridwanraji** [comment]
[Integrate Default Slide Template with Authoring view](https://trello.com/c/WiuqepF3/21-integrate-default-slide-template-with-authoring-view)

**Comment by ridwanraji** [comment]
@peggyopal The previous screenshots were from google chrome on a Apple Macbook.
Find below more screenshots for more devices.
screen shot 2018-03-04 at 3 18 42 pm
screen shot 2018-03-04 at 3 18 09 pm
screen shot 2018-03-04 at 3 17 42 pm

File: src/components/Designer.vue [hunk 1]
```diff
@@ -83,6 +83,20 @@
...
+              sm12
+              md12
+              lg12
+              xl12>
+              <form>
+                <font-style
+                class="my-0 py-0"/>
+              </form>
+            </v-flex>
+          </v-layout>
```
Comment (KingMackle) [inline src/components/Designer.vue]: Line 86-99 is this needed for this task. I can see this being useful when developing locally, but I don't think that it needs to pushed  to development or master .

Comment (KingMackle) [reply src/components/Designer.vue]: Moreover, there is a merge conflict that needs to be resolved. I think undoing this file will resolve it

Comment (peggyopal) [reply src/components/Designer.vue]: LOL I was literally fixing it when you commented 😛 Is it okay to be merged?

File: test/unit/specs/FontStyle.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import Vue from 'vue'
+import FontStyle from '@/components/FontStyle'
+
+// Test number of font colors showing
+describe('FontStyle.vue', () => {
+  it('Font Colors should be 11', () => {
+    const Constructor = Vue.extend(FontStyle)
+    const vm = new Constructor().$mount()
+    expect(vm.fontColors.length).to.equal(11)
```
Comment (peggyopal) [inline test/unit/specs/FontStyle.spec.js]: This should b `vm.$data.fontColor...` or something line that, we should be using data

Comment (ridwanraji) [reply test/unit/specs/FontStyle.spec.js]: I thought that was supposed to be for functions, test cases or PR added after the testing tutorial on wednesday?

**Pull request merged**